ca_gazebo/models/grey_square/model.sdf

```diff
@@ -0,0 +1,399 @@
+<?xml version='1.0'?>
+<sdf version='1.6'>
+  <model name='grey_square'>
```

Comment:
Consider calling it room/big_room or something like that.

Reply:
Adressed